indicators/serializers_new/participant_count_serializers.py

```diff
@@ -123,6 +124,93 @@ class Meta:
             'disaggregations'
         ]
 
+    def validate_date_collected(self, value):
+        """
+        Validates that date_collected is in the range of self.context.get('program').start_date and self.context.get('program').end_date
+
+        Params
+            value
+                - The value of date_collected
+
+        Raises
+            ValidationError
+                - If validation fails
+
+        Returns
+            Value of date_collected
+        """
+        program = self.context.get('program')
+
+        if program.reporting_period_start <= value <= program.reporting_period_end:
+            return value
+
+        raise exceptions.ValidationError('This date should be within the fiscal year of the reporting period.')
```

Comment:
Any of these error messages that might be displayed by the frontend should be wrapped in translation tags and be preceded by translator comments.

Reply:
Sounds good, I'll get those added.

Reply:
I was looking at commits for translator comments. Would a translator comment be along the lines of `# Translators: An error message detailing that the selected date must be within the fiscal years reporting periods`